src/ui-test/tests/15_commands.projects.new.output.folder.test.ts

```diff
@@ -0,0 +1,113 @@
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { afterEach } from 'mocha';
+import * as fs from 'node:fs';
+import path from 'path';
+import {
+	ActivityBar,
+	DefaultTreeSection,
+	EditorView,
+	InputBox,
+	ModalDialog,
+	ViewControl,
+	VSBrowser,
+	WebDriver,
+	Workbench
+} from 'vscode-extension-tester';
+import * as pjson from '../../../package.json';
+import {
+	CREATE_COMMAND_QUARKUS_ID,
+	expandItemInTree,
+	killTerminal,
+	SPECIFIC_WORKSPACE_NAME,
+	SPECIFIC_WORKSPACE_PATH,
+	TASK_FINISHED_IN_TERMINAL_TEXT,
+	waitUntilExtensionIsActivated,
+	waitUntilFileAvailable,
+	waitUntilTerminalHasText
+} from '../utils/testUtils';
+
+describe('Create a Camel Project in a new output folder', function () {
+	this.timeout(600000);
+	let driver: WebDriver;
+	let input: InputBox;
+
+	before(async function () {
+		driver = VSBrowser.instance.driver;
+		fs.mkdirSync(SPECIFIC_WORKSPACE_PATH, { recursive: true });
+	});
+
+	after(async function () {
+		await new EditorView().closeAllEditors();
+		await killTerminal();
+	});
+
+	const command = CREATE_COMMAND_QUARKUS_ID;
+
+
+	describe(`${command}`, function () {
+
+		beforeEach(async function () {
+			this.timeout(400000);
+			await VSBrowser.instance.openResources(SPECIFIC_WORKSPACE_PATH);
+			await VSBrowser.instance.waitForWorkbench();
+			await waitUntilExtensionIsActivated(driver, `${pjson.displayName}`);
+			await (await new ActivityBar().getViewControl('Explorer'))?.openView();
+		});
+
+		afterEach(async function () {
+			await new EditorView().closeAllEditors();
+			await killTerminal();
+		});
+
+		it(`Using command ${command}`, async function () {
+			const SUBFOLDER = `sub-folder-${CREATE_COMMAND_QUARKUS_ID}`;
+			const SUBFOLDER_FOLDER_PATH = path.join(SPECIFIC_WORKSPACE_PATH, SUBFOLDER);
+			fs.mkdirSync(SUBFOLDER_FOLDER_PATH, { recursive: true });
+
+			await new Workbench().executeCommand(command);
+
+			input = await InputBox.create(30000);
+			await input.setText('com.demo:test:1.0-SNAPSHOT');
+			await driver.sleep(1000);
+			await input.confirm();
+
+			input = await InputBox.create(30000);
+			const inputText = await input.getText();
+			await driver.sleep(1000);
+			await input.setText(path.join(inputText, SUBFOLDER));
+			await driver.sleep(1000);
```

Comment:
I do not understand why we would need a sleep between setting a text and clicking on confirm, the UI is not changing between the 2 actions

Reply:
Removed the sleeps after testing locally without them. Most of them where there while I was trying to figure out what was not working in the tests while not being able to attach the debugger to their execution. Lets see if the work inside github actions.